js/ui/codingManager.js

```diff
@@ -81,11 +81,19 @@ const CodingManager = new Lang.Class({
     },
 
     addBuilderWindow: function(actor) {
+        let tracker = Shell.WindowTracker.get_default();
+        let correspondingApp = tracker.get_window_app(actor.meta_window);
+        let isSpeedwagonForBuilder = (
+            Shell.WindowTracker.is_speedwagon_window(actor.meta_window) &&
+            (correspondingApp ? correspondingApp.get_id() === 'org.gnome.Builder.desktop' : false)
```

Comment:
Nitpick: I'd rather have a single AND of three conditions rather than using the ternary operator here.